includes/fbproduct.php

```diff
@@ -35,6 +35,7 @@ class WC_Facebook_Product {
 	const FB_VARIANT_IMAGE       = 'fb_image';
 	const FB_VISIBILITY          = 'fb_visibility';
 	const FB_REMOVE_FROM_SYNC    = 'fb_remove_from_sync';
+	const FB_RICH_TEXT_DESCRIPTION = 'fb_rich_text_description';
```

Comment:
Why do we need a variable for rich text description here and why we would want oto save it to the meta data? As we can populate it on the fly when needed from the woo data model. 
The "fb_product_description" is needed because we have a a text box in UI, so we have to save what users enters there. But in case of rich text description, we don't have it, unless we add UI of a rich text into existing "FB Description" text box.